contracts/src/L2StandardBridgeBot.sol

```diff
@@ -66,12 +68,30 @@ contract L2StandardBridgeBot is Ownable {
         withdrawTo(_l2Token, msg.sender, _amount, _minGasLimit, _extraData);
     }
 
-    // withdrawFee withdraw the delegation fee vault to _recipient address, only owner can call this function.
+    // withdrawFee withdraw the delegation fee vault to _recipient address on L2, only owner can call this function.
     function withdrawFee(address _recipient) external onlyOwner {
         (bool sent, ) = _recipient.call{ value: address(this).balance }("");
         require(sent, "Failed to send Ether");
     }
 
+    // withdrawFeeToL1 withdraw the delegation fee vault to _recipient address on L1, only owner can call this function.
+    //
+    // NOTE: Since the `withdrawTo` function is defined as external and `_extraData` is `bytes calldata`, the
+    // `_extraData` cannot be mock from `bytes memory` by the caller. See also https://github.com/ethereum/solidity/issues/12778
+    function withdrawFeeToL1(address _recipient, bytes calldata _extraData) external onlyOwner {
+        uint256 _balance = address(this).balance;
+        require(_balance > delegationFee, "fee vault balance is insufficient to pay the required amount");
+
+        uint256 _amount = _balance - delegationFee;
+        this.withdrawTo{ value: _balance }(
+            LEGACY_ERC20_ETH,
+            _recipient,
+            _amount,
+            160000 ,        // constant acceptable minGasLimit is fine
```

Comment:
better to leave it as a parameter in case the L1 fee used for this increased in the future

Reply:
Applied this suggestion at [e44879e](https://github.com/bnb-chain/opbnb-bridge-bot/pull/9/commits/e44879ebd3ae4f457469294bc69df9cd50330add)